scripts/aws/cfn/microservice.yml

```diff
@@ -377,6 +409,11 @@ Parameters:
     Description: 'Variable that defines whether to enable the simplified workflow'
     Default: 'false'
 
+  EnableTenderFeatureFlow:
+    Type: String
+    Description: 'Variable that defines whether to enable the new tender worflow'
+    Default: 'false'

```

Comment:
Giusto delle considerazioni:

1. Chiamarei la variabili `EnableSimplifiedTenderFlow`.
2. Metterei il default a true (se nell'SRS però c'è scritto il contrario, ignora questo punto).
3. Typo nella descrizione, worflow (manca la K).

Reply:
fixati tutti i punti. In SRS non è specificato il valore di default pertanto ho settato a true